### PR TITLE
Bug/161 add validations on form

### PR DESCRIPTION
**Pull request by tanmaythole**

### CODE REVIEW

File: src/Components/ProjectsComponent/PayloadComponents/AddNewPayload.jsx [hunk 1]
```diff
@@ -106,7 +116,7 @@ const AddNewPayload = (props) => {
               Expected Outcomes<span className="text-danger">*</span>
             </label>
             <Accordion>
-              {data?.expected_outcome?.map((item, index) => {
+              {expectedOutcome?.map((item, index) => {
                 return (
```
Comment (ChinmaySonawane) [inline src/Components/ProjectsComponent/PayloadComponents/AddNewPayload.jsx]: Use syntax of arrow function one line return

File: src/containers/ProjectContainer/HeaderContainer/index.jsx [hunk 2]
```diff
@@ -48,6 +49,20 @@ const HeaderContainer = (props) => {
...
+        name: name || "",
+        header: { ...header },
+        id: headerId || INITIAL_FORM_DATA.id,
+      }));
+    }
+    return () => setHeadersFormData({ ...INITIAL_FORM_DATA });
+  }, [selectedItem, cat]);
+
   const onFormDataChange = (e) => {
     setHeadersFormData((p) => ({
```
Comment (ChinmaySonawane) [inline src/containers/ProjectContainer/HeaderContainer/index.jsx]: use prev instead of p

Comment (tanmaythole) [reply src/containers/ProjectContainer/HeaderContainer/index.jsx]: 👍🏻

File: src/Components/ProjectsComponent/PayloadComponents/AddNewPayload.jsx (hunk 1 above)
Comment (tanmaythole) [reply src/Components/ProjectsComponent/PayloadComponents/AddNewPayload.jsx]: 👍🏻

**Review by ChinmaySonawane — APPROVED** [review]

**Pull request merged**